### ISSUE TITLE
[Monitoring v1] Hide v0.2.2 in Rancher v2.6

### ISSUE THREAD

**Issue by jiaqiluo** [rancher/rancher#32693]
**What kind of request is this (question/bug/enhancement/feature request):**
enhancement

The monitoring v1 chart version v0.2.2 should NOT show in the Rancher v2.6.x release.

**Comment by jiaqiluo** [comment]
Close this issue because it is not valid anymore. 
As of today, there is no new monitoring v1 chart version planned to be released for v2.5.9, so v0.2.2 needs to be available in v2.5.9

**Comment by sowmyav27** [comment]
Validate in rancher-images.txt also

**Comment by deniseschannon** [comment]
Available to test in v2.6.0-rc1

**Comment by cmurphy** [comment]
Validated on rancher/rancher:v2.6.0-rc1:

In Ember UI, navigated to Monitoring on local cluster, version 0.2.2 is not available:

![image](https://user-images.githubusercontent.com/1220710/128416682-e5fbd68b-0081-4fdd-94b9-910a2b9f613f.png)

Validate that rancher-images.txt only contains the mirrored versions of the images used in the chart:

```
$ curl -s -L https://github.com/rancher/rancher/releases/download/v2.6.0-rc1/rancher-images.txt | grep '\(coreos-prometheus-operator\|coreos-prometheus-config-reloader\|jimmidyson-configmap-reload\|prom-node-exporter\|grafana-grafana\|prom-prometheus\|thanosio-thanos\)'
rancher/mirrored-coreos-prometheus-config-reloader:v0.40.0
rancher/mirrored-coreos-prometheus-operator:v0.40.0
rancher/mirrored-grafana-grafana-image-renderer:3.0.1
rancher/mirrored-grafana-grafana:6.7.4
rancher/mirrored-grafana-grafana:7.5.8
rancher/mirrored-jimmidyson-configmap-reload:v0.3.0
rancher/mirrored-jimmidyson-configmap-reload:v0.4.0
rancher/mirrored-prom-node-exporter:v1.0.1
rancher/mirrored-prom-prometheus:v2.12.0
rancher/mirrored-prom-prometheus:v2.18.2
rancher/mirrored-thanosio-thanos:v0.21.1
```
